### ISSUE TITLE
Test: configuring web worker extension host affinity

### ISSUE THREAD

**Issue by joyceerhl** [microsoft/vscode#164515]
Refs: https://github.com/microsoft/vscode/issues/164048

- [x] anyOS @rebornix 
- [x] anyOS @IanMatthewHuff 

Complexity: 3

[Create Issue](https://github.com/microsoft/vscode/issues/new?body=Testing+%23164515%0A%0A&assignees=joyceerhl)

---

We have a setting `"extensions.experimental.affinity" which lets users configure specific extensions to run in a dedicated extension host, e.g. 

```json
  "extensions.experimental.affinity": {
    "GitHub.remotehub": 1,
    "vscode.github-authentication": 1,
    "vscode.git-base": 1
  },
```

This setting previously worked for local Node process extension hosts. Please now test that it works for web worker extension hosts as well:
1. Add this setting in vscode.dev for whichever extensions you would like to run in a dedicated extension host for improved performance. 
2. Verify in the web worker extension output channel that the configured extensions are placed on a separate extension host.
3. Report back with any extensions that benefit from this configuration--does it make a difference to performance e.g. for the [Vim extension](https://github.com/microsoft/vscode/issues/75627)

**Comment by IanMatthewHuff** [comment]
Seems good aside from the slightly confusing info messages. At the same time it was a bit harder to see if there were real perf improvements. With the setting off and on I tried a bunch of editing workflows. I only noticed one instance of what seemed like laggy typing with the setting off and none with it on, so maybe an improvement? Tricky bit was that I didn't get a consistent repro on the slow typing issue so not really enough data to be sure of anything.
